Generate docs from R API for distribution

Thanks!

Trying to follow along, stuck at
```
$ R -e "install.packages('roxygen2', repos='http://cran.rstudio.com/')"
...
clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/usr/local/opt/gettext/lib -L/usr/local/opt/readline/lib -L/usr/local/lib -L/usr/local/Cellar/r/3.4.1_2/lib/R/lib -L/usr/local/opt/gettext/lib -L/usr/local/opt/readline/lib -L/usr/local/lib -o xml2.so RcppExports.o connection.o xml2_doc.o xml2_init.o xml2_namespace.o xml2_node.o xml2_output.o xml2_schema.o xml2_url.o xml2_xpath.o -L/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk/usr/lib -lxml2 -lz -lpthread -licucore -lm -L/usr/local/Cellar/r/3.4.1_2/lib/R/lib -lR -lintl -Wl,-framework -Wl,CoreFoundation
ld: file not found: /usr/lib/system/libsystem_symptoms.dylib for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [xml2.so] Error 1
ERROR: compilation failed for package ‘xml2’
* removing ‘/usr/local/lib/R/3.4/site-library/xml2’
* installing *source* package ‘desc’ ...
** package ‘desc’ successfully unpacked and MD5 sums checked
** R
** inst
** preparing package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
* DONE (desc)
ERROR: dependency ‘xml2’ is not available for package ‘roxygen2’
* removing ‘/usr/local/lib/R/3.4/site-library/roxygen2’

The downloaded source packages are in
	‘/private/var/folders/m6/4yqn_4q129lbth_dq3qzj_8h0000gn/T/Rtmpcyo9eZ/downloaded_packages’
Warning messages:
1: In install.packages("roxygen2", repos = "http://cran.rstudio.com/") :
  installation of package ‘xml2’ had non-zero exit status
2: In install.packages("roxygen2", repos = "http://cran.rstudio.com/") :
  installation of package ‘roxygen2’ had non-zero exit status
> 
> 
```

--- a/adam-r/pom.xml
+++ b/adam-r/pom.xml
@@ -30,9 +30,10 @@
             </goals>
             <configuration>
               <executable>R</executable>
+              <workingDirectory>bdgenomics.adam</workingDirectory>
               <arguments>
                 <argument>-e</argument>
-                <argument>"library(devtools);devtools::document()"</argument>
+                <argument>library(devtools);devtools::document()</argument>
               </arguments>
             </configuration>
           </execution>

